api/v1/multiclusterhub_types.go

```diff
@@ -82,10 +82,6 @@ type MultiClusterHubSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Disable Update ClusterImageSets",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
 	DisableUpdateClusterImageSets bool `json:"disableUpdateClusterImageSets,omitempty"`
 
-	// (Deprecated) Enable cluster proxy addon
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Enable Cluster Proxy Addon",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
-	EnableClusterProxyAddon bool `json:"enableClusterProxyAddon,omitempty"`
```

Comment:
We cannot remove this field even though it is no longer used. It has to stay in the spec as deprecated